frontend/www/js/omegaup/components/problem/creator/cases/MultipleCasesInput.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="mt-3">
+    <b-row>
+      <b-col>
+        <b-form-group
+          data-prefix
+          :label="T.problemCreatorPrefix"
+          label-for="prefix"
+          class="mb-4"
+        >
+          <b-form-input v-model="multipleCasesPrefix" autocomplete="off" />
+        </b-form-group>
+      </b-col>
+      <b-col>
+        <b-form-group
+          :label="T.problemCreatorSuffix"
+          label-for="suffix"
+          class="mb-4"
+        >
+          <b-form-input v-model="multipleCasesSuffix" autocomplete="off" />
+        </b-form-group>
+      </b-col>
+    </b-row>
+    <b-form-group
+      :label="T.problemCreatorNumberOfCases"
+      :description="`${T.problemCreatorNumberOfCasesHelper} ${caseNamePreview}`"
+      label-for="case-points"
+    >
+      <b-form-input
+        v-model="multipleCasesCount"
+        type="number"
+        number
+        :min="0"
+      />
+    </b-form-group>
+    <b-form-group :label="T.problemCreatorGroupName" label-for="case-group">
+      <b-form-select v-model="multipleCasesGroup" :options="options" />
+    </b-form-group>
+  </div>
+</template>
+
+<script lang="ts">
+import { GroupID } from '@/js/omegaup/problem/creator/types';
+import { NIL } from 'uuid';
+import { Component, Vue } from 'vue-property-decorator';
+import T from '../../../../lang';
+
+@Component
+export default class MultipleCasesInput extends Vue {
+  multipleCasesPrefix = '';
+  multipleCasesSuffix = '';
+  multipleCasesCount = 0;
+  multipleCasesGroup: GroupID = NIL;
+
+  T = T;
+
+  get caseNamePreview() {
+    return `${this.multipleCasesPrefix + '1' + this.multipleCasesSuffix}, ${
```

Comment:
```suggestion
    return `${this.multipleCasesPrefix}1${this.multipleCasesSuffix}, ${
```

ya estamos en una cadena de interpolación, no hay necesidad de concatenar encima de eso.